lib/statsd.rb

```diff
@@ -165,28 +220,28 @@ def select_shard(stat)
     end
   end
 
-  def signed_payload(key, message)
-    sha256 = Statsd.setup_openssl
-    payload = timestamp + nonce + message
-    signature = OpenSSL::HMAC.digest(sha256, key, payload)
-    signature + payload
-  end
+  class Buffer
+    DEFAULT_BUFFER_CAP = 512
```

Comment:
Any reason for choosing this? Just curious if it is intentional or a best guess.